ceph_deploy/hosts/centos/install.py

```diff
@@ -68,35 +68,18 @@ def install(distro, version_kind, version, adjust_repos, **kw):
 
     if adjust_repos:
         if version_kind in ['stable', 'testing']:
-            distro.packager.add_repo_gpg_key(gpg.url(key))
-
-            if version_kind == 'stable':
-                url = 'https://download.ceph.com/rpm-{version}/{repo}/'.format(
-                    version=version,
-                    repo=repo_part,
-                    )
-            elif version_kind == 'testing':
-                url = 'https://download.ceph.com/rpm-testing/{repo}/'.format(repo=repo_part)
-
-            # remove any old ceph-release package from prevoius release
-            remoto.process.run(
-                distro.conn,
-                [
-                    'yum',
-                    'remove',
-                    '-y',
-                    'ceph-release'
-                ],
-            )
-            remoto.process.run(
-                distro.conn,
-                [
-                    'yum',
-                    'install',
-                    '-y',
-                    '{url}noarch/ceph-release-1-0.{dist}.noarch.rpm'.format(url=url, dist=dist),
-                ],
-            )
+            repo_suffix = 'el' if distro.normalized_name == 'eurolinux' else 'lp'
+            repo_mapping = {
+                'name': 'ceph-{suffix}'.format(suffix=repo_suffix),
+                'baseurl': 'https://repo.eurolinux.local/ceph/14_{suffix}/prod/'.format(suffix=repo_suffix),
+                'gpgcheck': '1',
+                # actually it's prefix but nvm
+                'gpgkey_url': 'https://repo.eurolinux.local/ceph/{suffix}_gpg.key'.format(suffix=repo_suffix),
+                'ssl_verify': '0'
+            }
+            yum_repo_text = '[{name}]\nname=Ceph {name}\nbaseurl={baseurl}' + \
+                            '\ngpgcheck={gpgcheck}\ngpgkey={gpgkey_url}\nsslverify={ssl_verify}'
+            distro.conn.remote_module.write_yum_repo(yum_repo_text)
```

Comment:
Wow, this section changed radically.
I'm not sure I understand correctly what's going on, but the lines removed added different repos for stable and testing, both hosted by *ceph*, then removed any preexisting version prior installing the new one.
The lines added doesn't do anything like this, but instead it seems that a new single repo is added (whether stable or testing) hosted by *eurolinux*, and there's no removal/installation of any package.

I'm not saying it's right or wrong, just that the functionality changed greatly.

Reply:
Sorry, but this one is wrong :). I forgot about this PR (since it was dead for like half year), and made some changes for our internal ceph-deploy. I will remove this PR.